src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-position: center;
+    height: 16px;
+    margin-right: 4px;
+    width: 16px;
+    background-repeat: no-repeat;
+  }
+
+  &.stars--4 :nth-child(-n + 4) {
```

Comment:
It should be the same functionality as in the `stars` task, if modifier is `--3` you should show 3 active stars as well